linux/cuda_info.bsh

```diff
@@ -80,10 +80,20 @@ source "${VSI_COMMON_DIR}/linux/versions.bsh"
 # .. seealso::
 #   :var:`CUDA_DEPRECATED`
 #
+# .. var:: CUDA_LTOS
+#
+# Starting in CUDA 11, NVIDIA introduced Link Time Optimizations (``lto_``) architectures in addition to the already existing ``sm_`` and ``compute_``
+#
+# Separately compiled code may not have as high of performance as whole program code because of the inability to inline code across files. A way to still get optimal performance is to use link-time optimization
```

Comment:
```suggestion
# Separately compiled kernels may not have as high of performance as if they were compiled together with the rest of the executable code because of the inability to inline code across files. Link-time optimization is a way to recover that improved performance
```